projects/uitdatabank/reference/entry.json

```diff
@@ -7438,6 +7438,71 @@
           "News articles"
         ]
       }
+    },
+    "/user": {
+      "get": {
+        "summary": "user - get",
+        "operationId": "user-get",
+        "description": "Returns the details of the current user.",
+        "tags": [
+          "Users"
+        ],
+        "responses": {
+          "200": {
+            "description": "The user details.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "uuid": {
+                      "type": "string",
+                      "example": "c8e6ff82-8b30-4295-b937-ab2f4f6ab4bf",
+                      "format": "uuid",
+                      "description": "UUID of the user."
+                    },
+                    "email": {
+                      "type": "string",
+                      "example": "info@publiq.be",
+                      "description": "The email address of the user."
+                    },
+                    "username": {
+                      "type": "string",
+                      "example": "username",
+                      "description": "The username of the user."
+                    }
```

Comment:
Someone who calls this endpoint will also get the old `id` and `nick` properties. To avoid confusion, I'd suggest to also document these but with [`"deprecated": true`](https://json-schema.org/draft/2020-12/json-schema-validation.html#section-9.3). That way they will get a deprecated flag in the docs and it will be clear that it's best to use the other ones instead. In the description you can specify which alternatives to use (`uuid` and `username` respectively)